FusionIIIT/applications/department/views.py

```diff
@@ -0,0 +1,149 @@
+import datetime
+import json
+from operator import or_
+from functools import reduce
+
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponse, HttpResponseRedirect
+# Create your views here.
+from django.db.models import Q
+from django.shortcuts import render
+from django.contrib.auth.models import User
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from applications.academic_information.models import Spi, Student
+from applications.globals.models import (Designation, ExtraInfo,
+                                         HoldsDesignation)
+from applications.eis.models import (faculty_about, emp_research_projects)
+
+#from .models import ()
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+
+# Create your views here.
+
+def index(request):
+    context = {}
+    print(request.user)
```

Comment:
Remember final code should not have any print statements or unnecessary comments. Use loggers(that too when there might be some errors, for example with try-except).